content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -219,30 +219,35 @@ sudo systemctl restart containerd
 ```
 {{% /tab %}}
 {{% tab name="Windows (PowerShell)" %}}
+
 ```powershell
 # (Install containerd)
-# download containerd
-cmd /c curl -OL https://github.com/containerd/containerd/releases/download/v1.4.1/containerd-1.4.1-windows-amd64.tar.gz
-cmd /c tar xvf .\containerd-1.4.1-windows-amd64.tar.gz
+# Download containerd
+curl.exe -L https://github.com/containerd/containerd/releases/download/v1.4.1/containerd-1.4.1-windows-amd64.tar.gz -o containerd-windows-amd64.tar.gz
+tar.exe xvf .\containerd-windows-amd64.tar.gz
```

Comment:
Probably a nit but is it worth changing it not be explicit on the version? 
```suggestion
curl.exe -L https://github.com/containerd/containerd/releases/download/v{version}/containerd-{version}-windows-amd64.tar.gz -o containerd-windows-amd64.tar.gz
tar.exe xvf .\containerd-windows-amd64.tar.gz
```

But other than that lgtm

Reply:
happy to be wrong on this! 😄

Reply:
the docs do something similar to this for cri-o: https://kubernetes.io/docs/setup/production-environment/container-runtimes/#cri-o

Reply:
I looked at setting this to the 'latest' release but that currently points to contaienrd v1.3.9 which won't work for windows here :(

Reply:
I did think of doing that tho!

Reply:
I can use {version} and add a comment that version should be >= v1.4

Reply:
updating - PTAL

Reply:
https://deploy-preview-26600--kubernetes-io-master-staging.netlify.app/docs/setup/production-environment/container-runtimes/#containerd

Reply:
Perfect!